k8s/models/node.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+from __future__ import absolute_import
+
+import six
+
+from .common import ObjectMeta
+from ..base import Model
+from ..fields import Field, OnceField, ListField
+
+
+class NodeSpec(Model):
+    externalID = OnceField(six.text_type)
+
+
+class NodeStatus(Model):
+    addresses = ListField(dict)
```

Comment:
I would prefer it if you could define the NodeAddress objects as well. While every item in the model could be described using a dict, the point of the models is to have some actual objects that can be introspected.

The same applies to NodeConditions and NodeSystemInfo below.